packages/angular/src/app/pages/planning-task-list/planning-task-list.component.scss

```diff
@@ -14,10 +14,8 @@
   }
 }
 
-::ng-deep {
+:host::ng-deep {
```

Comment:
May be ng-deep bound to .dx-toolbar class

Reply:
ok.. that's better